### DISCUSSION TITLE
traffic-analytics from readthedocs might be good to add it in the dashboard

### DISCUSSION THREAD

**Discussion by mxochicale** [SciKit-Surgery/sustainable-pkg-stats#34] [Ideas]
This might something to look at it https://readthedocs.org/dashboard/scikit-surgeryvtk/traffic-analytics/

![Screenshot from 2023-05-11 12-44-38](https://github.com/SciKit-Surgery/scikit-surgery-stats/assets/11370681/d9f69c40-b402-46f4-986d-322324e87215)

**Comment by yidilozdemir** [comment]
The link gives me a 404 error

**Comment by thompson318** [comment]
The link works for me, although I think you'll need to be logged in to read the docs with maintainer access to see it. Not sure if there's a way to automate access.
